loaders/authentication.js

```diff
@@ -0,0 +1,39 @@
+const fetch = require('node-fetch');
+const dotenv = require('dotenv').config();
```

Comment:
This only needs to be included once in the app.js

Reply:
So after including the library in app.js it gets included in all the other files?
Should I also move the node-fetch requirement?

Reply:
If I remove `const dotenv = require('dotenv').config();` when i try to get read the `UP_USER` and `UP_PWD` i get undefined that resolve in a Forbidden error from fetch.
So i would say that i cant remove it?

Reply:
I'll suggest a structure like in nijobs-be: https://github.com/NIAEFEUP/nijobs-be/blob/develop/src/config/env.js

`dotenv` only needs to be required, and it's config method called, once. Then, the values from the env file will be available in `process.env.*`.

I would recommend using `dotenv-flow` over `dotenv`. It may seem overkill for now, but being able to include `.env` files with default values as documentation is pretty awesome. It would also enable providing default values for testing and such, if we so desire, which is always a plus :slightly_smiling_face:

Reply:
@Andrepereira2001 After looking at the rest of the code I understand where you're coming from. Your approach was to run the authentication file directly and thus what @GambuzX said would not work.

However, this file is meant to be included and then the function ran. Since `app.js` is the back-end server's entrypoint, it will always be run first, and if `require('dotenv').config()` is placed in the top of the file, any subsequent code will be able to use the environment variables without issue.